src/components/ebay-carousel/examples/09-a11y-text/template.marko

```diff
@@ -11,10 +11,10 @@
     }
 </style>
 <ebay-carousel
-    items-per-slide="2"
     a11y-previous-text="Go to previous slide - Top Products"
     a11y-next-text="Go to next slide - Top Products"
-    a11y-status-text="Showing slide {currentSlide} of {totalSlides} - Top Products - Carousel"
+    a11y-title-text="Top Products"
+    a11y-status-text="{title} Showing slide {currentSlide} of {totalSlides} - Carousel"
```

Comment:
Will this create an unwanted space if `a11y-title-text` is not provided?

Reply:
Sure, but does that ever get seen, or read out by the screen reader?

Reply:
Screen readers do have very fine-grained punctuation settings. I'm honestly not sure if or how this might be treated, so I think its safest just to not have it if we can. Sorry.

Reply:
@ianmcburnie Adjusted. Let me know if the changes work.